### PR TITLE
JobSphere initialization

### PR DESCRIPTION
**Pull request by Sanober494**
This repo contains career guides, internship info, interview prep resources and more for undergrad students.

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -1,3 +1,172 @@
...
+
+The following are some global research internship positions:
+
+**CANADA**
+
+
+  * [University of Alberta Research Experience (UARE)](https://www.ualberta.ca/admissions-programs/visiting-student-and-internship-programs/research-internships/ualberta-research-experience/index.html)
+
+**US**
+  * [LPI Summer Intern Program in Planetary Science](https://www.lpi.usra.edu/lpiintern/application/), for both national and international students.
```
Comment (Fir121) [inline docs/index.md:37]: Recheck the formatting here, when served as a mkdocs site.

File: docs/index.md [hunk 2]
```diff
@@ -1,3 +1,172 @@
...
+**SWITZERLAND**
+
+  * [Research fellowship at ETH Zurich](https://www.inf.ethz.ch/studies/summer-research-fellowship.html) 
+
+  * [E3, EPFL (Switzerland)](https://eee.epfl.ch/)
+  * [Summer at EPFL](https://summer.epfl.ch/)
+
+
+**GERMANY**
+  * [Max Planck Institute for Gravitational Physics](https://www.aei.mpg.de/student-internships), for all students.
```
Comment (Fir121) [inline docs/index.md]: Here too. Adding a new line should fix it

File: docs/index.md [hunk 3]
```diff
@@ -1,3 +1,172 @@
...
+
+**TAIWAN**
+
+  * [NTHU Summer Internship Program](http://eng-en.web.nthu.edu.tw/files/14-1130-129169,r1447-1.php)
+  * [TIGP-IIP](https://tigpsip.apps.sinica.edu.tw/index.php), Academia Sincia, Taiwan
+  * [ITRI Global Internship program](https://www.itri.org.tw/english/ListStyle.aspx?DisplayStyle=05&SiteID=1&MmmID=617731531432246346)
+  * [TEEP@Asia (Taiwan)](https://teep.studyintaiwan.org/programs/Engineering)
+  * [TEEP Internship, Taiwan](https://www.roc-taiwan.org/in_en/post/2749.html)
+
+**JAPAN**
```
Comment (Fir121) [inline docs/index.md]: Newline after this

File: docs/index.md [hunk 4]
```diff
@@ -1,3 +1,172 @@
...
+  * [OIST Research Internship Program](https://admissions.oist.jp/oist-research-internship-program-description), Okinawa Institute of Science and Technology 
+  * [NIMS (Japan)](https://www.nims.go.jp/eng/hr-development/internship.html)
+  * [Hennge Global Internship Program](https://hennge.com/global/gip.html), for remote internship in a Japan-based tech start-up.
+  * [Internet Initiative Japan Internship] (https://www.iijlab.net/en/career/internship.html), for on-site tech internship in Japan.
+
+**AUSTRIA**
+
+  * [ISternship Summer Student Program](https://phd.pages.ist.ac.at/isternship/), IST Austria
+  
+**HONG KONG**
```
Comment (Fir121) [inline docs/index.md]: Newline here as well.

File: docs/index.md [hunk 5]
```diff
@@ -1,3 +1,172 @@
...
+## 2. Career Guides
+
+Land your dream job with confidence! The following career guides provide everything you need to build a strong resume, ace interviews, and launch your professional journey.
+
+ - [Resume and Cover Letters, issued by Harvard University](https://cdn-careerservices.fas.harvard.edu/wp-content/uploads/sites/161/2023/04/resume-and-letter_2022-1.pdf)
+ - [Career Handbook, issued by Massachusetts Institute of Technology](https://cdn.uconnectlabs.com/wp-content/uploads/sites/123/2021/07/Career-Handbook-2019.pdf)
+ - [Career Guide, issued by Stanford University](https://plusfive.sites.stanford.edu/sites/g/files/sbiybj20941/files/media/file/stanford_18-19_career_guide_plusfive_0.pdf)
+ - [my Career Path, issued by Carnegie Mellon University](https://www.cmu.edu/career/documents/career-guides/myCareerPathGuide2018-2019.pdf)
+
+## 3. Interview Preparation
```
Comment (Fir121) [inline docs/index.md]: Recheck this on the deployed version

**Review by Fir121 — CHANGES_REQUESTED** [review]
Overall awesome job, I'm excited to go through all the links but from the ones I saw very cool. Lmk when these small formatting changes are done, I will merge this.

File: docs/index.md [hunk 6]
```diff
@@ -1,3 +1,172 @@
...
+
+* **Career Development:** Craft a stellar resume using our library of templates and guides. Find valuable internship opportunities to gain real-world experience.
+* **Tech Interview Prep:** Conquer your technical interviews with our extensive collection of coding challenges. Explore solutions and explanations to hone your problem-solving skills.
+* **Centralized Resource:** No more juggling through countless websites. We strive to be your one-stop shop for all career-related information and preparation.
+
+**Get Started Today!**
+
+This is just the beginning. We're constantly adding new resources and features to help you navigate the exciting yet challenging world of professional careers. Dive deeper into the following sections to explore what we offer:
+
+1. [Research Internships](#research-internships)
```
Comment (Fir121) [inline docs/index.md:15]: Recheck these internal links

**Comment by Sanober494** [comment]
Fixed internal links and added newline

File: docs/index.md (hunk 1 above)
Comment (Fir121) [reply docs/index.md:37]: Use a newline, by doing this the output looks like:

image

It does not group with the other bullet points

**Comment by Fir121** [comment]
Looks good, still some unresolved though. Check it in the mkdocs site.

_Tried to setup an automatic PR previewer to make this easy but it doesn't seem to work for this repository so you will have to stick with local or with your own GitHub Pages to test it out._

**Comment by Fir121** [comment]
Also a note -> Deploy previews have been configured for all ACM open source projects! If you want an automatic preview you will have to make a new PR for the same code (will work for any new PRs from now on).

**Comment by Fir121** [comment]
Great work! One last thing, you can remove any extra `<br>` tags only use those when absolutely necessary, the extra space looks a bit odd in the preview.

Edit: haha adding `<br>` converted into html

**Comment by Fir121** [comment]
LGTM :rocket: awesome work

**Pull request merged**